Assets/Scripts/Entities/Units/Actor_Unit.cs

```diff
@@ -2158,7 +2158,12 @@ public void SubtractHealth(int damage)
         TurnsSinceLastDamage = -1;
     }
 
-    public bool Damage(int damage, bool spellDamage = false, bool canKill = true)
+    public int CalculateDamageWithResistance(int damage, DamageTypes damageType)
+    {
+        return 0;
```

Comment:
This was actually a fragment of an idea that I was testing the viability of, so not an official feature or anything for the PR in particular. It looks like I accidentally didn't undo these changes after scraping the idea,  since my typo fix and these changes were in the same file. Leading it to breaking damage, whoops.